cloudigrade/util/health.py

```diff
@@ -0,0 +1,20 @@
+"""Collection of custom backends to plug into ``django-health-check``."""
+import kombu
+from django.conf import settings
+from health_check.backends import BaseHealthCheckBackend
+
+
+class RabbitMQCheckBackend(BaseHealthCheckBackend):
+    """RabbitMQ Health Check."""
+
+    def check_status(self):
+        """Test RabbitMQ health check."""
+        conn = kombu.Connection(settings.RABBITMQ_URL)
+        try:
+            conn.connect()
```

Comment:
Given #210 (which _might_ be improved in a future release of Celery), this line might hang forever if RabbitMQ cannot be reached. As a newbie, I presume OpenShift will realize this is hanging and has its own timeout threshold that it will use to mark the service as critical/down, yes?

Reply:
Yes, the readiness/liveness probes themselves have a timeout.